app/components/manage-intermissions/index.hbs

```diff
@@ -33,6 +36,20 @@
           <tr>
             <td>{{moment-format intermission.startedAt "DD/MM/YYYY hh:mm"}}</td>
             <td>{{moment-format intermission.endedAt "DD/MM/YYYY hh:mm"}}</td>
+            <td>
+              {{#if intermission.position}}
+                {{#if (eq intermission.position "during")}}
+                  {{t "manageIntermissions.duringAp"}}
+                {{/if}}
+                {{#if (eq intermission.position "before")}}
+                  {{t "manageIntermissions.beforeAp"}}
+                {{/if}}
+                {{#if (eq intermission.position "after")}}
+                  {{t "manageIntermissions.afterAp"}}
+                {{/if}}
+                <strong>{{intermission.onderwerp.titel}}</strong>
```

Comment:
How hard would it be to make this a link to the specified behandeling? I saw as bold and instinctively I clicked it thinking it will bring be to the behandeling. Maybe we should discuss the feature but I think is a nice addition

Reply:
not trivial I think, we don't really do anything with url fragments yet
but I think it's a really good idea, also in the gebruikersessie we saw people clicking on various things expecting they were links

Reply:
But the side menu has the links, for example http://localhost:4200/meetings/60B9D8A10AAB420009000001/edit#behandeling-60B9D8CC0AAB420009000004 goes to the first behandeling, what do you mean with url fragments? the hashes?

Reply:
I think the clicking on this also has to do with the style of the bold, something to check with dieter I think

Reply:
it works as long as you remain on the same route, non trivial otherwise